homeassistant/components/season/sensor.py

```diff
@@ -100,31 +103,31 @@ def get_season(
         autumn_start = spring_start.replace(month=9)
         winter_start = spring_start.replace(month=12)
 
+    season = STATE_WINTER
     if spring_start <= current_date < summer_start:
         season = STATE_SPRING
     elif summer_start <= current_date < autumn_start:
         season = STATE_SUMMER
     elif autumn_start <= current_date < winter_start:
         season = STATE_AUTUMN
-    elif winter_start <= current_date or spring_start > current_date:
-        season = STATE_WINTER
```

Comment:
I tend to prefer all values inside the condition rather than presetting. But it's just my personal preference.

Reply:
The condition itself is useless, so it would become an `else` in that case.
Which... should not be needed.